x-aeson/ambiata-x-aeson.cabal

```diff
@@ -15,7 +15,11 @@ library
                        aeson                           >= 0.8        && < 0.12
                      , base                            >= 3          && < 6
                      , bytestring                      == 0.10.*
+                     , containers                      == 0.5.*
                      , text                            >= 1.1        && < 1.3
+                     , time                            >= 1.5        && < 2
```

Comment:
Also, we probably shouldn't be using `time` in new libraries, should really use `thyme` instead. Not sure if it's a good idea to have combinators in here encouraging use of `time`.